CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* Clarify that dot-keys define tables and when they stop accepting new keys.
```

Comment:
Clarify that dotted keys implicitly define tables.
Clarify behavior of tables defined implicitly by dotted keys.